lib/xcpretty/formatters/simple.rb

```diff
@@ -8,12 +8,14 @@ class Simple < Formatter
     PASS = "✓"
     FAIL = "✗"
     PENDING = "⧖"
+    MEASURE = "⧗"
 
     ASCII_PASS = "."
     ASCII_FAIL = "x"
     COMPLETION = "▸"
     ASCII_PENDING = "P"
     ASCII_COMPLETION = ">"
+    ASCII_MEASURE = "T"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.